src/server/pfs/fuse/filesystem_test.go

```diff
@@ -419,19 +497,26 @@ func testFuse(
 	require.NoError(t, err)
 	apiClient := pfsclient.NewAPIClient(clientConn)
 	mounter := fuse.NewMounter(localAddress, apiClient)
-
+	fmt.Printf("XXX created clientConn\n")
 	mountpoint := filepath.Join(tmp, "mnt")
 	require.NoError(t, os.Mkdir(mountpoint, 0700))
 	ready := make(chan bool)
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
+		fmt.Printf("XXX mounting\n")
 		require.NoError(t, mounter.Mount(mountpoint, nil, nil, ready))
 	}()
 
 	<-ready
+	fmt.Printf("XXX mounted\n")
+
 	defer func() {
+		fmt.Printf("XXX Trying to unmount\n")
 		_ = mounter.Unmount(mountpoint)
+		fmt.Printf("XXX Unmounted!\n")
 	}()
+	fmt.Printf("XXX running test callback\n")
 	test(apiClient, mountpoint)
+	fmt.Printf("XXX ran test callback\n")
```

Comment:
I think if there's an intermittent issue we're trying to track down leaving printfs in a test can be fine since they shouldn't be seen most of the time when tests pass and there's not much chance of a user running into them. On the other hand we may hit a point when there's too many things being printed at which point it'll stop being useful and we'll have to cull them. But we can deal with that when we get there.